src/main/java/io/cdap/e2e/pages/locators/CdfSignInLocator.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.locators;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * Represents CdfSignInLocators
+ */
+public class CdfSignInLocator {
+
+    @FindBy(how = How.XPATH, using = "//input[@name='identifier']")
+    public WebElement cdfUsername;
+
+    @FindBy(how = How.XPATH, using = "//input[@name='Passwd']")
+    public WebElement cdfPassword;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(text(),'Next')]//parent::button")
+    public WebElement nextButton;
+
+    public static By locatorOfEmailTextBox() {
+        return By.xpath("//input[@name='identifier']");
+    }
+
+    public By selectTestAccount(String testAccountName) {
+        return By.xpath("//*[contains(text(),'" + testAccountName + "')]");
+    }
+
+    public By clickOnAllowButton() {
+        return By.xpath("//button[contains(text(),'Allow')]");
+    }
+
+    public By clickOnContinueButton() {
+        return By.xpath("//span[contains(text(),'Continue')]");
+    }
+
+    public static By locatePluginNameInList(String pluginName, String  pluginGroupName) {
```

Comment:
This locator is already present in `CdfStudioLocators`, can we reuse it??

Reply:
Commented: https://github.com/cdapio/cdap-e2e-tests/pull/197#discussion_r1307602850